src/Certify.Core/Management/Challenges/DNSChallengeHelper.cs

```diff
@@ -52,9 +53,15 @@ public class DNSChallengeHelper
                         var azureDns = new DnsProviderCloudflare(credentials);
                         dnsAPIProvider = azureDns;
                     }
+
+                    if (providerDefinition.Id == "DNS01.API.GoDaddy")
+                    {
+                        var azureDns = new DnsProviderGoDaddy(credentials);
```

Comment:
Let's rename azureDns to goDaddyDns or just merge this into one line with no var.